### PR TITLE
Enhancement: Add error message for adjustment creation if user balance is insufficient

### PR DESCRIPTION
**Pull request by Kohelbekker**

### CODE REVIEW

File: app/api/v2/admin/adjustments.rb [hunk 1]
```diff
@@ -128,6 +128,11 @@ class Adjustments < Grape::API
           post '/action' do
             admin_authorize! :update, Adjustment
             adjustment = Adjustment.find(params[:id])
+            member_balance = Member.find_by(uid: adjustment.receiving_account_number.split('-')[2]).get_account(adjustment.currency).balance
+
+            if adjustment.amount < 0 && adjustment.amount.abs() > member_balance
+              error!({ errors: ['admin.adjustment.negative_adjustment_bigger_than_member_balance'] }, 422)
```
Comment (Kohelbekker) [inline app/api/v2/admin/adjustments.rb]: Error looks too long, but I haven't found shorter sentence which is clear enough.

File: app/api/v2/admin/adjustments.rb [hunk 2]
```diff
@@ -128,6 +128,11 @@ class Adjustments < Grape::API
           post '/action' do
             admin_authorize! :update, Adjustment
             adjustment = Adjustment.find(params[:id])
+            member_balance = Member.find_by(uid: adjustment.receiving_account_number.split('-')[2]).get_account(adjustment.currency).balance
```
Comment (Kohelbekker) [inline app/api/v2/admin/adjustments.rb]: Maybe, there is shorter way to get member's balance which I haven't found.

Comment (mnaichuk) [reply app/api/v2/admin/adjustments.rb]: better to do smth like this:
account_number_hash = Operations.split_account_number(account_number: adjustment.receiving_account_number)
member = Member.find_by(uid: account_number_hash[:member_uid]
balance = member.get_account(account_number_hash[:currency_id]

and better to move this logic inside if like:
if adjustment.amount < 0
  `Select user, balance...`
  `Check if enough balance`
end

File: app/api/v2/admin/adjustments.rb (hunk 1 above)
Comment (mnaichuk) [reply app/api/v2/admin/adjustments.rb]: can be like `admin.adjustment.user_insufficient_balance`

**Review by mnaichuk — CHANGES_REQUESTED** [review]

**Review by mnaichuk — APPROVED** [review]

**Pull request merged**